ExpenseClaimListActivityTests/AndroidManifest.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
So somehow a separate project got created just for these tests. The test case class should be inside the same package as the other tests (inside `Team9ProjectTests`), otherwise they won't run with the rest of the tests.